test/gtest/common/test_obj_size.cc

```diff
@@ -44,7 +44,7 @@ UCS_TEST_F(test_obj_size, size) {
    UCS_TEST_SKIP_R("Statistic enabled");
 #else
     EXPECTED_SIZE(ucp_ep_t, 104);
-    EXPECTED_SIZE(ucp_request_t, 200);
+    EXPECTED_SIZE(ucp_request_t, 224);
```

Comment:
did we increase request object size? if so how come tests didn't fail previously?

Reply:
i guess need to run this in jenkins in release mode ?

Reply:
right, I don't know why it is not launched in regular testing...

Reply:
@hoopoepg pls, in this PR, need to make sure this test runs in jenkins in release mode

Reply:
@yosefe done